just-pandas-things.ipynb

```diff
@@ -535,12 +535,12 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/Users/chip/miniconda3/envs/py3/lib/python3.6/site-packages/ipykernel_launcher.py:2: SettingWithCopyWarning: \n",
+      "<ipython-input-12-fa76da25894c>:2: SettingWithCopyWarning: \n",
```

Comment:
let's not re-run cells you didn't change so the results aren't overwritten

Reply:
Made sure of this in PRs #8 and #9. I apologize. I hadn't checked the diff until after your comment.